news/_posts/core-weekly/2018-09-27-coreweekly-week-38-2018.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 38 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2018-09-27 15:15:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 17th of August to Sunday 23rd of September 2018.
```

Comment:
Did you mean `from Monday 17th of September`

Reply:
:-(